src/main/archetype/dispatcher.ams/src/conf.d/proxy/cif-magento-graphql.proxy

```diff
@@ -0,0 +1,9 @@
+
+Include /etc/httpd/conf.d/variables/cif-magento-proxy.vars
+
+# Defines the proxy rule to access the Magento GraphQL endpoint
+
+<IfDefine ENABLE_CIF_MAGENTO_PROXY>
+	ProxyPass "/api/graphql" "${MAGENTO_HOST}/graphql" max=20 connectiontimeout=5 timeout=5 ttl=30
```

Comment:
if you want to remove cookies, you need to wrap the proxy directives in a Location
this would be replaced with something like:
```
<IfDefine ENABLE_CIF_MAGENTO_PROXY>
    <Location "/api/graphql">
        ProxyPass "${MAGENTO_HOST}/graphql" max=20 connectiontimeout=5 timeout=5 ttl=30
        <IfModule mod_headers.c>
            RequestHeader unset Cookie
            Header unset Set-Cookie
        </IfModule>
    </Location>
</IfDefine>
```